### PR TITLE
Remove duplicate attributes from CoreLib that are in CoreFX

### PR DESCRIPTION
**Pull request by shrah**
Couldn't remove LCIDConversionAttribute as it is being used in mscorlib.

@jkotas @tijoytom

### CODE REVIEW

File: src/mscorlib/src/System/Internal.cs [hunk 1]
```diff
@@ -25,7 +25,6 @@
 [assembly:Guid("BED7F4EA-1A96-11d2-8F08-00A0C9A6186D")]
 
 // The following attribute are required to ensure COM compatibility.
-[assembly:System.Runtime.InteropServices.ComCompatibleVersion(1, 0, 3300, 0)]
 [assembly:System.Runtime.InteropServices.TypeLibVersion(2, 4)]
```
Comment (jkotas) [inline src/mscorlib/src/System/Internal.cs]: You can delete the Guid and TypeLibVersion too while you are on it,

**Comment by jkotas** [comment]
> Couldn't remove LCIDConversionAttribute as it is being used in mscorlib.

So it should be delete in corefx then. 

While you are on it, I think you can also delete these from here and move them to corefx:

```
TypeLibImportClassAttribute 
ProgIdAttribute 
ImportedFromTypeLibAttribute 
ComEventInterfaceAttribute 
TypeLibVersionAttribute 
```

File: src/mscorlib/src/System/Internal.cs (hunk 1 above)
Comment (shrah) [inline src/mscorlib/src/System/Internal.cs]: Done.

**Comment by shrah** [comment]
@jkotas, couldn't remove ProgIdAttribute as it is being used in mscorlib. I have removed the rest of the attributes.

**Comment by jkotas** [comment]
@dotnet-bot test Windows_NT x64 Debug Build and Test please

**Comment by stephentoub** [comment]
@jkotas, @shrah, it looks like the removal of ComEventInterfaceAttribute is breaking https://github.com/dotnet/corefx/pull/15779

**Pull request merged**